docs/application/dotnet/get-started/wearable/first-xaml-app.md

```diff
@@ -126,6 +130,30 @@ If you face any issues during deployment, it is recommended to manually install
 ```bash
 $ sdb install <path-to-package>/org.tizen.example.TizenWearableXamlApp1-1.0.0.tpk
 ```
+## Validate Your Application Changes Using XAML Hot Reload
+
+To test your UI code changes in your application, you would have to build and deploy your application. With the introduction of the XAML Hot Reload feature, you can now immediately see the UI edits on the running emulator without having to rebuild or redeploy your app. This feature helps you save time and validate your changes easily.
```

Comment:
app->application

Reply:
@sijupunnoose,

Thank you for your review. I will do the needful.